README.md

```diff
@@ -1,90 +1,94 @@
-    - On the **Definition** tab, if there are problems with the **Locations** entries, click **Reload**
-    - Click **Set as Target Platform** in the top-right corner
-
-At this point you should be all set, unless Eclipse is not able to generate protobuf sources.
-Following the explanations [here](https://github.com/trustin/os-maven-plugin) may help.
-As a workaround, you can run `mvn compile` on the command line to generate protobuf sources,
-and in Eclipse hit `F5` on the project with build errors.
-
-### Eclipse quirks
-
-Strange issues in Eclipse and their remedies.
-
-- Sometimes, for no apparent reason, the generated protobuf classes may disappear,
-  and Eclipse may report compilation errors when resolving these class names, symbols.
-  The workaround is to run `mvn compile` and refresh the views in Eclipse (click on the project and press `F5`).
-
-Running
--------
-
-Open `plugin.xml` of `org.sonarline.eclipse.core` (for example), and see the **Run** and **Debug** buttons in the top-right corner.
-
-Running tests
--------------
-
-In Eclipse:
-
-1. Create a run configuration by running a test class first with **Run As... / JUnit Plug-in Test**
-
-2. Edit the configuration
-
-    - On the **Test** tab, uncheck **Run in UI thread**
-    - On the **Main** tab, under **Program to Run**, select **Run an application** with value **[No Application] - Headless Mode**
-
-With Maven:
-
-    mvn clean verify
-
-Adding a dependency
--------------------
-
-Must be osgi bundle.
-
-### For Maven
-
-Add the artifact to the parent pom.
-
-Run `mvn compile` to get the artifact downloaded so that you can inspect its manifest in the jar.
-
-Find the name of the bundle from its manifest, see the `Bundle-SymbolicName` property.
-
-Edit the manifest of the package where you want to add the dependency (for example: `org.sonarlint.eclipse.core/META-INF/MANIFEST.MF`), add the bundle in the `Require-Bundle` property, using its symbolic name.
-
-If the bundle is not needed at runtime, don't forget to mark it optional, to avoid including in the package.
-(Edit properties on the **Dependencies** tab, or append `;resolution:=optional`)
-
-To verify the content of the package: `mvn clean package` and check content of the ZIP in plugins folder.
-
-### For Eclipse
-
-Add to `target-platform-e46/target-platform-e46-dev.target` (or whatever target you use) the path of the artifact in your local maven repository as a filesystem path, similar to already existing dependencies.
-
-In the target editor (or open `plugin.xml`), click **Set as Target Platform**.
-Note that this will trigger a compilation in Eclipse.
-
-At this point, and if the artifact exists at the specified path, it should be usable, and Eclipse will be able to compile the project.
-
-### License
-
-Copyright 2015-2017 SonarSource.
-
-Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.gnu.org/licenses/lgpl.txt)
+SonarLint for Eclipse
+=============
+
+## Installing and Using

```

Comment:
Note that `## title` is the same as `title` and `----` on the next line, producing h2 headings. Since other parts in the file use the second style, please follow that pattern.

More importantly, looking at the diff of this pull request, it looks as if all lines have been changed. Ignoring line endings, you only added this one section. So it seems you committed the file with DOS line endings instead of UNIX. Can you please fix that?